build.gradle.kts

```diff
@@ -0,0 +1,45 @@
+plugins {
+    `java-library`
+    kotlin("jvm") version "1.9.0"
+    kotlin("plugin.allopen") version "1.9.10"
+    kotlin("plugin.spring") version "1.9.10"
+    kotlin("plugin.noarg") version "1.9.10"
+    kotlin("plugin.jpa") version "1.9.10"
```

Comment:
why do u need `noarg` and `jpa`?

Reply:
noarg for no argument constructor (because hibernate needs one)
jpa for noarg so it see all his annotations